src/components/ErrataInfo.vue

```diff
@@ -320,6 +320,13 @@
             </q-card-section>
             <q-card-actions align="right">
                 <q-btn no-caps color="green" @click="updateAdvisory(advisory.id)" :loading="loading">Save</q-btn>
+                <q-btn
+                  v-if="errataStatuses.RELEASED === advisory.release_status"
```

Comment:
Sorry for the pedantic comment, but I would prefer to read `advisory.release_status === errataStatuses.RELEASED`